RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.cc

```diff
@@ -215,7 +222,44 @@ MeasurementTrackerEventProducer::updateStrips( const edm::Event& event, StMeasur
   }
 }
 
+//FIXME: just a temporary solution for phase2!
 void 
+MeasurementTrackerEventProducer::updatePhase2OT( const edm::Event& event, Phase2OTMeasurementDetSet & thePh2OTDets ) const {
+
+
+  // Phase2OT Clusters
+  if (pset_.existsAs<std::string>("Phase2TrackerCluster1DProducer")) {
```

Comment:
... it's not the first instance in this class, but still.
pset_ shouldn't be checked every event. Better save a relevant parameter flag and use it.